xbmc/cores/RetroPlayer/rendering/RenderContext.h

```diff
@@ -33,6 +41,22 @@ class CRenderSystemBase;
 class CWinSystemBase;
 class TransformMatrix;
 
+enum GL_SHADER_METHOD
```

Comment:
`enum class` to avoid polluting the namespace with short, possibly ambiguous constants